EventsExpress/ClientApp/src/components/header-profile/header-profile.js

```diff
@@ -64,13 +64,14 @@ export default class HeaderProfile extends Component {
                                     </Tooltip>
 
                                 </Link>
-                                <div>
-                                    <button className="btn btn-outline-secondary" onClick={this.handleClick}>
-                                        <i className="fas fa-plus mr-1"></i>
-                                    add event
-
-                                    </button>
-                                </div>
+                                {id && role == 'User' && (
```

Comment:
Let's create a separate component for doing role comparisons that we can wrap required elements with.